From 7e98cb22209917798b84317aacd0f6b84cc05416 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:01:40 -0400
Subject: [usb] core/hcd: replaced C99 // comments

Message-id: <1410887463-20674-29-git-send-email-dzickus@redhat.com>
Patchwork-id: 95373
O-Subject: [RHEL7 PATCH 028/591] drivers: usb: core: hcd: replaced C99 // comments
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 048cb21c37a917b4872de9cd004cbf017a65b362
Author: Matthias Beyer <mail@beyermatthias.de>
Date:   Sat Oct 5 18:02:08 2013 +0200

    drivers: usb: core: hcd: replaced C99 // comments

    Signed-off-by: Matthias Beyer <mail@beyermatthias.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/core/hcd.c b/drivers/usb/core/hcd.c
index 138ed82..b98f3d9 100644
--- a/drivers/usb/core/hcd.c
+++ b/drivers/usb/core/hcd.c
@@ -428,7 +428,7 @@ rh_string(int id, struct usb_hcd const *hcd, u8 *data, unsigned len)
  char const *s;
  static char const langids[4] = {4, USB_DT_STRING, 0x09, 0x04};
 
- // language ids
+ /* language ids */
  switch (id) {
  case 0:
   /* Array of LANGID codes (0x0409 is MSFT-speak for "en-us") */
@@ -615,7 +615,7 @@ static int rh_call_control (struct usb_hcd *hcd, struct urb *urb)
  case DeviceOutRequest | USB_REQ_SET_INTERFACE:
   break;
  case DeviceOutRequest | USB_REQ_SET_ADDRESS:
-  // wValue == urb->dev->devaddr
+  /* wValue == urb->dev->devaddr */
   dev_dbg (hcd->self.controller, "root hub device address %d\n",
    wValue);
   break;
@@ -625,7 +625,7 @@ static int rh_call_control (struct usb_hcd *hcd, struct urb *urb)
  /* ENDPOINT REQUESTS */
 
  case EndpointRequest | USB_REQ_GET_STATUS:
-  // ENDPOINT_HALT flag
+  /* ENDPOINT_HALT flag */
   tbuf[0] = 0;
   tbuf[1] = 0;
   len = 2;
@@ -683,7 +683,7 @@ error:
   if (urb->transfer_buffer_length < len)
    len = urb->transfer_buffer_length;
   urb->actual_length = len;
-  // always USB_DIR_IN, toward host
+  /* always USB_DIR_IN, toward host */
   memcpy (ubuf, bufp, len);
 
   /* report whether RH hardware supports remote wakeup */
@@ -1135,7 +1135,7 @@ long usb_calc_bus_time (int speed, int is_input, int isoc, int bytecount)
    return (9107L + BW_HOST_DELAY + tmp);
   }
  case USB_SPEED_HIGH: /* ISOC or INTR */
-  // FIXME adjust for input vs output
+  /* FIXME adjust for input vs output */
   if (isoc)
    tmp = HS_NSECS_ISO (bytecount);
   else
-- 
1.7.1